cauldron/redis_cache.py

```diff
@@ -107,6 +107,22 @@ def get_key(cls, key, namespace=None):
                 key = cls._get_key(namespace, key)
             return (yield from redis.get(key, encoding=cls._utf8))
 
+    @classmethod
+    @coroutine
+    def sadd(cls, key, value, namespace=None, *values):
```

Comment:
Can't give non keyword argument after keyword argument. This will break when given arguments 1,1,namespace=2,1
and for the other values we have to add namespace
Check delete_keys method in this file